core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -609,16 +609,37 @@ void removeProperty(String tableName, String property)
    * Gets properties of a table. This operation is asynchronous and eventually consistent. It is not
    * guaranteed that all tablets in a table will return the same values. Within a few seconds
    * without another change, all tablets in a table should be consistent. The clone table feature
-   * can be used if consistency is required.
+   * can be used if consistency is required. Method calls getConfiguration and then calls
```

Comment:
Could add a javadoc links to the `getConfiguration()` method here and in `NamespaceOperations`.